app/views/shared/_plausible.html.erb

```diff
@@ -1,2 +1,2 @@
-<!-- change to script.js when going live --!>
 <script defer data-domain="orcid.princeton.edu" src="https://plausible.io/js/script.local.js"></script>
+<script>window.plausible = window.plausible || function() { (window.plausible.q = window.plausible.q || []).push(arguments) }</script>
```

Comment:
@JaymeeH what does this do?  Maybe a comment on why it is needed?

Reply:
There were errors saying that Plausible was undefined when attempting to send the plausible event. This is what was in pdc plausible file, so I copied it over and that got rid of the errors we were getting. If there's a better way around it, please let me know